packages/vite/src/client/client.ts

```diff
@@ -345,6 +345,13 @@ export function updateStyle(id: string, content: string): void {
     style.setAttribute('data-vite-dev-id', id)
     style.textContent = content
 
+    const cspNonce = document.querySelector<HTMLMetaElement>(
+      'meta[property=csp-nonce]',
+    )?.content
```

Comment:
Does `cspNonce` need to change during dev? I think we should cache it if not to avoid doing the query every time we update a style.